test/topology_custom/test_commitlog_segment_data_resurrection.py

```diff
@@ -0,0 +1,124 @@
+from test.pylib.manager_client import ManagerClient
```

Comment:
Please add license/copyright blurb.

Reply:
It should be copyrighted by you, not ScyllaDB, since you wrote it.

Reply:
could you show any example in scylla files please?

Reply:
if my work based on @denesb code should I mention him in copyright as well?

Reply:
Right, this test is copied from dtest (for which I'm the author), then modified by @alezzqz (to resolve review comments).

Reply:
Ok

Reply:
@denesb then you get to review it

Reply:
Why should I review my own code? :)

In any case, @nyh already reviewed this, the only thing left to sort out is the copyright paperwork.
As far as I'm concerned this PR is good to go in.